factories/exporters.go

```diff
@@ -86,6 +87,7 @@ var defaultExporters = []exporter.Factory{
 	loggingexporter.NewFactory(),
 	logzioexporter.NewFactory(),
 	lokiexporter.NewFactory(),
+	nopexporter.NewFactory(),
```

Comment:
I'm guessing this likely conflicts with the nop exporter we have in this list, I think you'll have to remove that one:
`exportertest.NewNopFactory()`

Reply:
Yeah that's what I'm finding too.